Doc/using/cmdline.rst

```diff
@@ -407,6 +407,8 @@ Miscellaneous options
    * ``-X showalloccount`` to output the total count of allocated objects for
      each type when the program finishes. This only works when Python was built with
      ``COUNT_ALLOCS`` defined.
+   * ``-X importprofile`` to show how long each import takes. Nested imports are
+     indented. It can be used to optimize application's startup time.
```

Comment:
Hum, maybe rename the option to "-X importime"? I don't know.

Maybe explain that the displayed time is the *cumulative* time, including import time of sub-imports.

Reply:
I changed importprofile to importtime.
And *self* time is shown now, as Serhiy's suggestion.